platform/ui/src/components/userPreferencesForm/UserPreferencesForm.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
I think we should check for effect dependencies, we should refactor state or the hook instead of disabling hook rules

Reply:
In this case there are no hooks defined so this comment can be removed.